diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter2/KdocWarnTest.kt

```diff
@@ -203,4 +203,89 @@ class KdocWarnTest : LintTestBase(::KdocComments) {
                 """.trimMargin()
         )
     }
+
+    @Test
+    @Tag(WarningNames.KDOC_NO_CONSTRUCTOR_PROPERTY)
+    fun `check simple primary constructor with comment`() {
+        lintMethod(
+                """
+                    |/**
+                    | * @property name d
+                    | * @param adsf
+                    | * @return something
+                    | */
+                    |class Example constructor (
+                    |   // short
+                    |   val name: String
+                    |) {
+                    |}
+                """.trimMargin(),
+                LintError(7, 4, ruleId, "${KDOC_NO_CONSTRUCTOR_PROPERTY.warnText()} // short", true)
+        )
+    }
+
+    @Test
+    @Tag(WarningNames.KDOC_NO_CONSTRUCTOR_PROPERTY)
+    fun `shouldn't trigger because not primary constructor`() {
+        lintMethod(
+                """
+                    |/**
+                    | * @property name d
+                    | * @property anotherName text
+                    | */
+                    |class Example {
+                    |   constructor(
+                    |   // name
+                    |   name: String,
+                    |   anotherName: String,
+                    |   OneMoreName: String
+                    |   )
+                    |}
+                """.trimMargin()
+        )
+    }
+
+    @Test
+    @Tag(WarningNames.KDOC_NO_CONSTRUCTOR_PROPERTY)
+    fun `check another constructor`() {
+        lintMethod(
+                """
+                    |/**
+                    | * @return some
+                    | */
+                    |class Example (
+                    |   //some descriptions
+                    |   name: String,
```

Comment:
This should be marked as `val` or `var` to be class property, ordinary constructor parameters (not properties) should not be checked by this inspection.